db/dbtypes/types.go

```diff
@@ -1649,6 +1649,22 @@ type Tx struct {
 	IsMainchainBlock bool `json:"mainchain"`
 }
 
+// AtomicSwapInfo describes the contract and redemption txs of an atomicswap.
+type AtomicSwapInfo struct {
+	Contract         string  `json:"contract"`
+	ContractTxID     string  `json:"contract_tx_id"`
+	ContractAddress  string  `json:"contract_address"`
+	ContractValue    float64 `json:"contract_value"`
+	FormattedAmount  string  `json:"formatted_amount"`
+	RecipientAddress string  `json:"recipient_address"`
+	RefundAddress    string  `json:"refund_address"`
+	Locktime         string  `json:"locktime"`
+
+	RedemptionTxID      string   `json:"redemption_tx_id"`
+	RedeemedToAddresses []string `json:"redeemed_to_addresses"`
+	IsRefundToInitiator bool     `json:"is_refund_to_initiator"`
+}
```

Comment:
I'd prefer this in `api/types` since the data isn't coming from or being used by the DB, but also I want to limit the use of `dbtypes` to other `db/...` packages as much as possible in the future.

Reply:
This is done but it feels wrong having `explorer` reference `apitypes`.

Reply:
I understand the feeling, but it's definitely good for dbtypes.  Both `explorer` and `api` need the `AtomicSwapInfo` (now `AtomicSwap`) type.  I'm open to other suggestions, but I believe an effort in 6.0 will be to get down to maybe 1 types package.  ref: https://github.com/decred/dcrdata/issues/1725

Reply:
oops, meant to say **not** good for dbtypes